apps/registry/pages/api/jobs.js

```
@@ -55,35 +54,75 @@ export default async function handler(req, res) {
-      similarity: documents[index].similarity,
+      similarity: doc ? doc.similarity : 0,
     };
   });
 
-  const filteredJobs = sortedJobs.filter(
+  // Get date distribution
+  const validDates = jobsWithSimilarity
+    .map((job) => {
+      const date = new Date(job.created_at);
+      // Check if date is valid
+      if (isNaN(date.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)